src/main/java/org/seqdoop/hadoop_bam/BAMInputFormat.java

```diff
@@ -186,7 +207,13 @@ static Path getIdxPath(Path path) {
 		return splits.stream()
 				.filter(split -> !((FileSplit) split).getPath().getName().endsWith(
 						SplittingBAMIndexer.OUTPUT_FILE_EXTENSION))
+                                .filter(split -> !((FileSplit) split).getPath().getName().endsWith(
+                                                BAMIndex.BAMIndexSuffix))
 				.collect(Collectors.toList());
+        }
+    
+    	static Path getBAIPath(Path path) {
+		return path.suffix(BAMIndex.BAMIndexSuffix);
```

Comment:
We also need to cover the case for files named `file.bai` as well as `file.bam.bai`.